projects/testing-library/tests/render.spec.ts

```diff
@@ -296,3 +297,49 @@ describe('DebugElement', () => {
     expect(view.debugElement.componentInstance).toBeInstanceOf(FixtureComponent);
   });
 });
+
+describe('initialRoute', () => {
+  @Component({
+    standalone: true,
+    selector: 'atl-fixture2',
+    template: `<button>Secondary Component</button>`,
+  })
+  class SecondaryFixtureComponent {}
+
+  @Component({
+    standalone: true,
+    selector: 'atl-router-fixture',
+    template: `<router-outlet></router-outlet>`,
+    imports: [RouterModule],
+  })
+  class RouterFixtureComponent {}
+
+  @Injectable()
+  class FixtureResolver implements Resolve<void> {
+    public isResolved = false;
+
+    public resolve() {
+      this.isResolved = true;
+    }
+  }
+
+  it('allows initially rendering a specific route to avoid triggering a resolver for the default route', async () => {
+    const expectedRoute = 'correct-route';
+    const routes = [
+      { path: expectedRoute, component: FixtureComponent },
+      { path: '**', resolve: { data: FixtureResolver }, component: SecondaryFixtureComponent },
+    ];
+
+    await render(RouterFixtureComponent, {
+      initialRoute: expectedRoute,
+      routes,
+      detectChangesOnRender: false,
```

Comment:
Is there a reason why detect changes is set to false?

```suggestion
```

Reply:
Mainly just to isolate the test from any extra logic, but I can remove it since it doesn't add any value here